integration/sawtooth_integration/tests/test_events_and_receipts.py

```diff
@@ -180,13 +180,17 @@ def assert_unsubscribe_response(self, msg):
             events_pb2.ClientEventsUnsubscribeResponse.OK)
 
 class BatchSubmitter:
-    def __init__(self):
+    def __init__(self, timeout):
         self.batches = []
         self.imf = IntkeyMessageFactory()
+        self.timeout=timeout
```

Comment:
This is a small problem. The '=' should have whitespace on both sides. Our test files aren't linted, but it is best not to introduce more lint.